### ISSUE TITLE
Not Able Play Shoutcast Streams !

### ISSUE THREAD

**Issue by hope-atlast** [brave/brave-browser#9486]
Not Able Play Shoutcast Streams !
![Screenshot from 2020-04-27 18-35-43](https://user-images.githubusercontent.com/29015880/80375495-0b1b8600-88b6-11ea-9406-8e9be07424ad.png)

**Comment by bsclifton** [comment]
@hope-atlast all of the MP3 ones work just fine for me. I don't know if Chromium has bundled AAC support. Are you not able to play the MP3 ones?

**Comment by hope-atlast** [comment]
Let me tell you , I even went to settings and Reset them. But still, all the channels, show that same message.

**Comment by rebron** [comment]
@hope-atlast Wondering if this is shields related? If you turn shields off, does it work for you?

**Comment by hope-atlast** [comment]
No, shields are off, but still . . .

![image](https://user-images.githubusercontent.com/29015880/80459057-6d749500-894f-11ea-9ef4-bb5473bf44b5.png)

**Comment by bsclifton** [comment]
@hope-atlast MP3 and other formats are not free formats... I think you need to install support for them.

Can you please try installing `chromium-codecs-ffmpeg-extra` and `chromium-codecs-ffmpeg-nonfree` and then quit/relaunch Brave please?

**Comment by hope-atlast** [comment]
Already Present

![image](https://user-images.githubusercontent.com/29015880/80476297-d2d48000-8967-11ea-9655-ec2840da7e02.png)

**Comment by Martina-Neumayer** [comment]
Same in my Arch. Brave is unable to play any stream on this page.
Firefox in contrary works without any problems.

System: Arch, kernel v.5.6.13-arch1-1 x86_64
Brave: Version 1.10.60 Chromium: 81.0.4044.138 (Official Build) unknown (64-bit)
Firefox: 76.0.1

**Comment by rebron** [comment]
cc: @srirambv to take a look

**Comment by Martina-Neumayer** [comment]
Now works but only when I go to the website trough "http". If I go with the "https" does not work.

System: Arch, kernel v.5.6.15-arch1-1 x86_64
Brave: Version 1.10.85 Chromium: 81.0.4044.138 (Official Build) unknown (64-bit)

![2020-06-06_21-07](https://user-images.githubusercontent.com/13028529/83952532-c16d7600-a839-11ea-9f67-1879fc76818f.png)

**Comment by rebron** [comment]
@Martina-Neumayer Did you use Shields to toggle `Connections upgraded to HTTPS` to make this work?

**Comment by Martina-Neumayer** [comment]
@rebron No.. I did nothing with Shields. They are as they were by default after the Brave installation.
I tried this option on and off now as You mentioned it, but without any effect.
